### ISSUE TITLE
Clarification on model training 

### ISSUE THREAD

**Issue by jonas-nothnagel** [iPieter/bert-corona-tweets#1]
Hi guys! Great work!

Last year I was supporting a project working on the same topic but using English Tweets from Turkey. 

I just read your paper, but would like to get more technical information about the model training. Is there additional info available? 

1. Is the topic classification is a Multi-label classification model? 
2. As far as I understood, you use the pre-trained MBert model and fine-tune it on your labeled dataset? The stored models on HF are  those trained models, right?
3. On what GPU did you train the models? 

Thanks a lot :)

**Comment by iPieter** [comment]
Thanks for the interest in our work! It's really cool to see that these questions arise in other countries as well. 

To answer your questions:
1. Although we have a multi-label dataset, only a small fraction of tweets is about multiple topics (140 chars is not a lot after all). So we used multiclass classification for the topic model. 
2. Yes 🙂 
3. A 1080 Ti

I'll close this issue. If you have any more questions, feel free to re-open it 🙂
